projects/pedal_board/inc/pedal_data.h

```diff
@@ -0,0 +1,17 @@
+#pragma once
+#include "ads1015.h"
+#include "pedal_calib.h"
+
+typedef struct PedalDataTxStorage {
+  Ads1015Channel brake_channel;
+  Ads1015Channel throttle_channel1;
+  Ads1015Channel throttle_channel2;
+} PedalDataTxStorage;
+
+StatusCode pedal_data_init(Ads1015Storage *storage, PedalCalibBlob *calib_blob);
+
+Ads1015Storage *get_ads1015_storage();
+
+PedalDataTxStorage *get_pedal_data_storage();
+
+PedalCalibBlob *get_pedal_calib_blob();
```

Comment:
Okay now I understand why you save calib blob here... Change the name of this module to PedalSharedResourcesProvider. Then this will be the module that provides PedalCalibBlob and Ads1015Storage. You can get rid of the PedalDataTxStorage definition and just have three #defines that define brake_channel, and throttle_channel's. You can put them in the header, and use them in pedal_data, and throttle_data respectively.